### PR TITLE
Implement LaTeX support

### PR DESCRIPTION
**Pull request by NicusorN5**
In these two commits, I've configured MkDocs so you can render LaTeX  snippets inside the pages.

Example picture:
screenshot

### CODE REVIEW

File: docs/about.md.bak [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# About UO Studio
```
Comment (deccer) [inline docs/about.md.bak]: I think this .bak file needs to go.

Would you mind adding it to `.gitignore`?

**Review by deccer — CHANGES_REQUESTED** [review]
Ah hehe, you removed it already.

If you can just tell .gitignore

then it LGTM

File: docs/about.md.bak (hunk 1 above)
Comment (NicusorN5) [reply docs/about.md.bak]: Didn't I remove it already? lol

I will modify the `.gitignore` file for future changes.

Comment (NicusorN5) [reply docs/about.md.bak]: Done!

**Review by deccer — APPROVED** [review]

**Pull request merged**